datasets/us/ice_wanted/crawler.py

```diff
@@ -0,0 +1,153 @@
+from zavod import Context
+from xml.etree import ElementTree
+from normality import collapse_spaces
+
+
+def get_element_text(doc: ElementTree, xpath_value: str, to_remove=[]) -> str:
+    """Extract text from each child nodes of an xpath and joins them together
+
+    Args:
+        doc (ElementTree): HTML Tree
+        xpath_value (str):  xpath to extract text from
+        to_remove (list, optional): string to remove in the extracted text.
+    """
+    element_tags = doc.xpath(xpath_value)
+
+    tag_list = []
+    for tag in element_tags:
+        tag_list.append(tag.text_content())
+
+    element_text = "".join(tag_list)
+
+    for string in to_remove:
+        element_text = element_text.replace(string, "")
+
+    return collapse_spaces(element_text.strip())
+
+
+def generate_mw_notes(**kwargs):
+    note_string = f"""Additional Information:
+- Wanted For: {kwargs.get('crime_summary')}
+- Wanted Heading: {kwargs.get('crime_details')}
```

Comment:
we don't really define or render this as markdown and we don't render newlines on the website. What could work nicely is if this was a list of strings without the dash instead. Then each attribute would be separated and easy to render in reasonable ways for everyone.

Reply:
and the descriptive attributes in the description property, while the category, status, crime etc can go in the notes property

Reply:
Got it